### PR TITLE
Add go-report widget

### PR DESCRIPTION
**Pull request by yuva29**
Signed-off-by: Yuva Shankar <yuva29@users.noreply.github.com>

### CODE REVIEW

**Comment by gaurav-dalvi** [comment]
LGTM

**Comment by rchirakk** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-[![Build Status](http://contiv-ci.ngrok.io/job/Netplugin%20Push%20Build%20Master/badge/icon)](http://contiv-ci.ngrok.io/job/Netplugin%20Push%20Build%20Master/)
+[![Build Status](http://contiv-ci.ngrok.io/job/Netplugin%20Push%20Build%20Master/badge/icon)](http://contiv-ci.ngrok.io/job/Netplugin%20Push%20Build%20Master/)
```
Comment (unclejack) [inline README.md]: These should be HTTPS URLs.

Comment (dseevr) [inline README.md]: Nice catch 👍

Comment (yuva29) [inline README.md]: Done!

**Pull request merged**